prov.py

```diff
@@ -2,19 +2,50 @@
 import random
 import locale
 import os
+import re, json, sys
+from urllib2 import Request, urlopen, URLError
+from bs4 import BeautifulSoup
 
 import polyglot
 from polyglot.text import Text, Word
 
+adress_dict={"fr":"https://fr.wiktionary.org/w/api.php?titles=Annexe:Liste_de_proverbes_fran%C3%A7ais"
+,"en":"https://en.wiktionary.org/w/api.php?titles=Appendix:English_proverbs"}
+
+def get_list(lang="fr"):
+    if not lang == "fr" and not lang == "en":
+        print "Only support fr & en languages"
```

Comment:
Can you use the print() for python3 here ?

Reply:
Done